packages/workbox-precaching/models/PrecachedDetailsModel.mjs

```diff
@@ -31,10 +31,13 @@ class PrecachedDetailsModel {
   /**
    * Construct a new model for a specific cache.
    *
+   * @param {string} dbName
    * @private
    */
-  constructor() {
-    this._db = new DBWrapper(`workbox-precaching`, 2, {
+  constructor(dbName) {
```

Comment:
Just wanted to double-check: it looks like the existing codebase [is already passing in](https://github.com/GoogleChrome/workbox/blob/f5792fd9c130fbd7916c7b3763752cf795d9b8a5/packages/workbox-precaching/controllers/PrecacheController.mjs#L44) a string parameter when calling this constructor, even though it was previously being ignored. So now the change is that you're using that string value inside of this constructor.

Reply:
Exactly.